doc/builtin_generators.rst

```diff
@@ -29,21 +29,21 @@ command-line interface (CLI)::
     usage: stone [-h] [-v] [--clean-build] [-f FILTER_BY_ROUTE_ATTR]
                  [-w WHITELIST_NAMESPACE_ROUTES | -b BLACKLIST_NAMESPACE_ROUTES]
                  generator output [spec [spec ...]]
-    
```

Comment:
I configured Visual Studio Code to auto-remove extraneous whitespace at EOL. Sorry for cluttering up the PR! lmk if I should reintroduce this whitespace.